Cabal/Distribution/Compat/ReadP.hs

```diff
@@ -297,6 +298,11 @@ skipSpaces =
   skip (c:s) | isSpace c = do _ <- get; skip s
   skip _                 = do return ()
 
+skipSpaces1 :: ReadP r ()
+-- ^ Like 'skipSpaces' but succeeds only if there is at least one
```

Comment:
As I understand it, Distribution.Compat.ReadP is a compatibility module which was verbatim copied some time ago from Text.ParserCombinators.ReadP in the base package. It uses a different code formatting style from the main cabal code. Incidentally, what is the reason why we need this compatibility module rather than just have cabal depend on ReadP from base as a library?